HelpSource/scdoc.css

```diff
@@ -15,10 +17,9 @@ html {
 }
 
 div.contents {
-    margin: 1em;
     padding-left: 0.3em;
     padding-right: 0.3em;
-    padding-top: 1.5em;
+    padding-top: 1.4em;
```

Comment:
i'm not big on this. the top "SuperCollider 3.9dev Classes" looks cramped against the top bar.